src/qtcore/qml/QMLColor.js

```diff
@@ -1,5 +1,14 @@
 // TODO
 function QMLColor(val) {
-    return val;
+  if(typeof(val) === "number") {
+    // we assume it is int value and must be converted to css hex with padding
+    // http://stackoverflow.com/questions/57803/how-to-convert-decimal-to-hex-in-javascript
+    val = "#" + (Math.round(val) + 0x1000000).toString(16).substr(-6).toUpperCase();
+  } else
+  if(typeof(val) === "array" && val.length >= 3) {
+    // array like [r,g,b] where r,g,b are in 0..1 range
+    var m = 255;
+    val = "rgb(" + Math.round(m * val[0]) + "," + Math.round(m * val[1]) + "," + Math.round(m * val[2]) + ")";
+  }
+  return val;
 }
-
```

Comment:
Could you leave the newline in place for now? Let's minimize the unnecessary changes to avoid conflicts.
We could clean things up later, after merging.